administration/src/mui-modules/application/primitive-inputs/CheckboxForm.tsx

```diff
@@ -39,12 +39,7 @@ const CheckboxForm: Form<State, ValidatedInput, AdditionalProps, Options> = {
                 }}
               />
             }
-            label={
-              <>
-                {label}
-                {options.required ? ' *' : ''}
-              </>
-            }
+            label={<>{label}</>}
```

Comment:
```suggestion
            label={label}
```
Is there any issue with that?

Reply:
no, just got confused a bit